cumulusci/robotframework/Salesforce.py

```diff
@@ -1235,11 +1237,35 @@ def _check_for_login_failure(self):
             return True
         return False
 
-    def elapsed_time_for_last_record(self, obj_name, start_field, end_field, **kwargs):
+    def elapsed_time_for_last_record(
+        self, obj_name, start_field, end_field, order_by, **kwargs
+    ):
+        """Compare a record's start-time to its end-time to see how long a process took.
```

Comment:
This terminology is unfamiliar to me. What does it mean for an object to have a start time and end time? Is that true for all objects, or just some types of objects? For example, does a Contact have a start time and end time?

Reply:
Perhaps its clearer now?

        """For records representing jobs or processes, compare the record's start-time to its end-time to see how long a process took."""

Reply:
That is definitely better.

Should there be a check and a human-readable error if you pass in the wrong type of object?

Reply:
The code doesn't really know if you've passed in the right or wrong type of object. All that matters is whether it can parse the two date fields. I've added nicer error messages around those: 0d1c8a2e .